src/ProxyClient/Nginx.php

```diff
@@ -109,7 +84,7 @@ private function buildPurgeUrl($url)
             $pathStartAt = strpos($url, $urlParts['path']);
             $purgeUrl = substr($url, 0, $pathStartAt).$this->purgeLocation.substr($url, $pathStartAt);
         } else {
-            $purgeUrl = $this->getBaseUrl().$this->purgeLocation.$url;
+            $purgeUrl = $this->purgeLocation.$url;
```

Comment:
the base url is now in the http adapter, right? otherwise this could be a problem in some situations